mealy/error_analyzer.py

```diff
@@ -474,13 +487,13 @@ def inverse_transform_thresholds(self):
         return self._error_clf_thresholds
 
     # TODO: rewrite this method using the ranking arrays
-    def error_node_summary(self, leaf_selector='all_errors', add_path_to_leaves=False, print_summary=False):
+    def error_node_summary(self, leaf_selector='all', add_path_to_leaves=False, print_summary=False):
```

Comment:
Why is this method called `error_node_summary` if it can only return summary from leaves? Likewise, why do we have the doc saying `Return summary information regarding input nodes.`? Might be a bit confusing

Reply:
I renamed it as `leaf_node_summary` and clarified the text.